Makefile

```diff
@@ -8,11 +8,23 @@ tests: lint ## run the tests
 	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs --junitxml=python_junit.xml --cov-report=xml --cov-branch
 	yarn test
 
-lint: ## run linter
+lintjs: ## run linter
+	yarn lint
+
+lintpy: ## run linter
 	flake8 jupyterfs setup.py
+
+lints: ## run linter
```

Comment:
change to `lint` for consistency (e.g. with `nbcelltests`)

Reply:
see comment on `fixes`